tools/CMakeLists.txt

```diff
@@ -4,7 +4,11 @@
 
 ADD_LIBRARY (toolstuff STATIC common_write.c)
 
-SET (tools_list mergesolv dumpsolv installcheck testsolv repo2solv)
+IF (WIN32)
+    INCLUDE_DIRECTORIES (${PROJECT_SOURCE_DIR}/win32/)
+ENDIF ()
+
+SET (tools_list testsolv)
```

Comment:
What's with the other tools? `dumpsolv` is pretty useful to have. And the others don't hurt either.

Reply:
I was successful in compiling all other default tools except repo2solv.

Additionally, enabling the `-DDEBIAN` flag now also works on windows.